finance/accounting.md

```diff
@@ -1,52 +1,68 @@
 # Accounting data
 
 Accounting data describes the money that we **know** we have spent or that we **highly expect** to receive.
-There are a few kinds of accounting data we have access to:
 
-All 2i2c-specific accounting documents should be accessible on our Team Drive to anybody with a `@2i2c.org` address.
-We don't control the access conditions for the CS&S documents described below, so if you don't have access to these, ask the {role}`Executive Director`.
+## Accounting transactions airtable
 
-We split our financial and accounting data between two places:
+{term}`CS&S` provides us a monthly dataset of the last 6 months of all of all our accounting transactions.
+This is [in a Google Sheet that is automatically updated each day][gsheet].
+It includes the last 6 months of any transaction 2i2c has made (if we need more than 6 months of history, ask `fsp@2i2c.org` for a custom data dump).
 
```

Comment:
I am not able to access the Google sheet that is automatically updated. I sent a request for access through Google.

Reply:
Right now the sheet is "owned" by CS&S, and it's only got a sym-link in our CS&S shared drive. I'm not sure if we can move the sheet there or not but in the meantime you should be able to get access via Joe. Let me know if you can't